src/Microsoft.Azure.WebJobs.Host/Executors/JobHostContext.cs

```diff
@@ -17,20 +18,26 @@ internal sealed class JobHostContext : IDisposable
         private readonly IListener _listener;
         private readonly TraceWriter _trace;
         private readonly IAsyncCollector<FunctionInstanceLogEntry> _fastLogger; // optional
+        private readonly IAsyncCollector<FunctionResultLog> _aggregator;
```

Comment:
I wanted them to live SxS for the time being. I didn't want to require it to be a "choose one logging strategy" just yet.